annotationPipeline/pom.xml

```diff
@@ -12,6 +12,10 @@
     <version>1.0.0</version>
   </parent>
 
+  <properties>
+    <start-class>org.cbioportal.annotation.AnnotationPipeline</start-class>
```

Comment:
Seems like this is required to prevent maven build failure after adding the generated client files.

Reply:
@onursumer do we still need this now that we are importing the client files from java api client repo?

Reply:
I think we can remove this one too

Reply:
@inodb seems like we still need this for some reason, let's keep it for now.